telegraf.conf

```diff
@@ -247,11 +247,11 @@
 
 [[inputs.exec]]
   commands = [
-      "curl -s 127.0.0.1:14002/api/dashboard", # Open SNO API by mapping ports when running your SNO docker instance
-      "curl -s 127.0.0.1:14003/api/dashboard" # If multiple nodes, map ports accordingly
+      "curl -s 127.0.0.1:14002/api/sno", # Open SNO API by mapping ports when running your SNO docker instance
+      "curl -s 127.0.0.1:14003/api/sno" # If multiple nodes, map ports accordingly

```

Comment:
URL should end with a / or curl needs the -L to follow redirects.

Reply:
i can confirm this